contents/docs/api/index.mdx

```diff
@@ -106,6 +106,19 @@ Sometimes requests are paginated. If that's the case, it'll be in the following
 
 You can then just call the `"next"` URL to get the next set of results.
 
+## Rate Limiting
+
+All requests to PostHog, except for evaluating feature flags and sending events (i.e. the [POST-only public endpoints](/docs/api/post-only-endpoints)), are rate limited.
+
+There's separate limits for different kinds of resources.
+
+For all CRUD endpoints, the rate limits are `480/minute` and `4800/hour`. 
+
+For all analytics endpoints (like calculating insights, retrieving persons, retrieving session recordings), the rate limits are `240/minute` and `1200/hour`.
```

Comment:
```suggestion
For all analytics endpoints (such as calculating insights, retrieving persons, retrieving session recordings), the rate limits are `240/minute` and `1200/hour`.
```

Nit: These are examples of endpoints, not things which are similar to endpoints.